### PR TITLE
Add a command dependency and its first sub-command endpoint

### PR DESCRIPTION
**Pull request by JarvisG495**
swctl dependency endpoint [endpoint-id] [--start=start-time] [--end=end-time]

Query the dependencies of an endpoint and show results in JSON format.

### CODE REVIEW

**Comment by wu-sheng** [comment]
Interesting adding. I think service dependency should be more important and widely used. It is better to add it too.

**Comment by JarvisG495** [comment]
> Interesting adding. I think service dependency should be more important and widely used. It is better to add it too.

thx! I am going to add it.

**Comment by JarvisG495** [comment]
I refactor the dependency command for further development.
Now the original version of dependency becomes dependency's sub-command: endpoint.
endpoint and service(TODO) will act as two dependency's sub-commands.

File: internal/commands/dependency/dependency.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+package dependency
+
+import (
+	"github.com/urfave/cli"
+)
+
+var Command = cli.Command{
+	Name:      "dependency",
+	ShortName: "dp",
```
Comment (fgksgf) [inline internal/commands/dependency/dependency.go:26]: I prefer `dep`

**Review by fgksgf — COMMENTED** [review]
LGTM

**Review by kezhenxu94 — APPROVED** [review]

File: internal/commands/dependency/dependency.go (hunk 1 above)
Comment (fgksgf) [reply internal/commands/dependency/dependency.go:26]:
```suggestion
	ShortName: "dep",
```

**Review by fgksgf — APPROVED** [review]

**Pull request merged**